pkg/validation/external/removed_apis.go

```diff
@@ -0,0 +1,43 @@
+package external
+
+import (
+	"github.com/operator-framework/api/pkg/manifests"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func GetRemovedAPIsOn1_25From(bundle *manifests.Bundle) map[string][]string {
+	deprecatedAPIs := make(map[string][]string)
+	for _, obj := range bundle.Objects {
```

Comment:
This check has not been called in any validator so it will not be useful. 
See that we would need to ensure that it will be called via the operatorhub and the alpha-deprecated one. 

we also need to address improvements for these scenario checks such as make clear what are the k8s versions supported by the checks and some cleanups.